src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random generator = new Random();
```

Comment:
Initialization of complex objects should by made in the constructor. This way you can be sure when the actual object is created.